src/__tests__/validationUtils.test.ts

```diff
@@ -20,3 +20,19 @@ describe("validatePostalCode", () => {
     expect(validatePostalCode("56789")).toBe(false);
   });
 });
+
+describe("validateAFM", () => {
+  it("returns false for invalid afms", () => {
+    expect(validateAFM("12345678")).toBe(false);
+    expect(validateAFM("xxxxxxxx")).toBe(false);
+    expect(validateAFM("141212176")).toBe(false);
+    expect(validateAFM("111111111")).toBe(false);
+  });
+  it("returns false for valid afms", () => {
```

Comment:
The test name should be `"returns true for valid afms"`

Reply:
😓 Fixed as well as the types.